### PR TITLE
Print largest assets included

### PR DESCRIPTION
**Pull request by aussiegeek**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -25,6 +25,27 @@ if (!testing) {
...
+      });
+
+      console.log('\n\n20 largest assets');
+
+      for (var i=0;i<20;i++) {
+        var module = modules[i];
+        console.log(module.name + ' ' + module.size);
+      }
+
+      console.log();
```
Comment (nelix) [inline webpack.config.js]: hmm?

Comment (nelix) [inline webpack.config.js]: new line or something?

**Comment by nelix** [comment]
Very cool. :+1:

File: webpack.config.js (hunk 1 above)
Comment (aussiegeek) [inline webpack.config.js]: Yeah printing a newline

**Pull request merged**